gradle.properties-example

```
@@ -42,6 +42,10 @@ wc.screenshots.url = example.com
 wc.screenshots.username = user@example.com
 wc.screenshots.password = password
 
+wc.e2e.real.api.url = example.com
+wc.e2e.real.api.email = user@example.com
+wc.e2e.real.api.password = password
+
```

The dummy values had to be added to `gradle.properties-example` to avoid complaints (e.g. see fail in 86b9ce0715ce47304404975624b09de1cba1d397) in cases when this file is used for `gradle.properties` (via [cp gradle.properties-example gradle.properties](https://github.com/woocommerce/woocommerce-android/blob/add-support-for-real-API-ui-tests/.buildkite/pipeline.yml#L66))